### ISSUE TITLE
[A11y] Fix A11y issues

### ISSUE THREAD

**Issue by gbowne1** [gbowne1/gbowne1site_react#40]
Issues are contrast, keyboard and text labeling and are hitting these issues:

- [ ] Clickable elements must be focusable and should have interactive semantics
- [ ] Content with images must be labeled
- [ ] Focusable element may be missing focus styling.
- [ ] Does not meet WCAG standards for accessible text
- [ ] Interactive elements must be labeled.

**Comment by adeleke5140** [comment]
Hi @gbowne1 I have experience fixing a11y bugs, do you need some help on this?

**Comment by gbowne1** [comment]
Ok @adeleke5140 I will assign you

**Comment by adeleke5140** [comment]
Thanks.

**Comment by adeleke5140** [comment]
I cloned your project locally and ran `npm start`. What I get is a blank page on `localhost:3000`, by any chance @gbowne1, do you know what the problem could be?

<img width="1411" alt="Screenshot 2023-10-04 at 18 23 49" src="https://github.com/gbowne1/gbowne1site_react/assets/32024147/c294b432-aa54-4e0e-ac06-934ef5a18f00">

**Comment by gbowne1** [comment]
@adeleke5140 

This app/site is messed up completely 

I don't know why.  There is a navigation bar and menu at the top but it has gotten all messed up.  At one point it was rendering fine, but after a push/merge  that was made  it no longer didnt.. and I have not been able to figure out what broke it.

**Comment by gbowne1** [comment]
If you hover over blank areas of the blue and white area in places, you might find theres hidden items like links.

The navbar on desktop should be no more than 80-100px high and I preferred 65px high.

**Comment by gbowne1** [comment]
There is a comment in issue #34 that' show what it should look like.

**Comment by adeleke5140** [comment]
I noticed there are a lot of bugs, some if it are caused by `MUI` but we would have to potentially dig deeper to fix it.

**Comment by gbowne1** [comment]
Yeah this app has a number of issues.  Feel free to make or file any issues in the issues tab on GitHub that you find.

**Comment by adeleke5140** [comment]
Got it. I just pushed a PR for the `a11y` errors. 

Would it be possible for you to add the `hacktoberfest` label to the issues. It could also incentive other people to help out.

**Comment by gbowne1** [comment]
I am not in front of my PC at the moment,  but will do so as soon as am able.

My Primary goal is to work on CodeBooker and ReactSocialNetwork aka reactsocialnetwork.

https://github.com/gbowne1/reactsocialnetwork and https://github.com/gbowne1/CodeBooker

**Comment by gbowne1** [comment]
@adeleke5140 I went and added the label to CodeBooker and ReactSocialNetwork and I also reviewed and merged your PR.  Will look deeper when I get back to the PC later today.

**Comment by adeleke5140** [comment]
That's great. I will take a look at the other repos as well.
